docs/context.rst

```diff
@@ -11,8 +11,11 @@ You can access them by importing them from ``connexion.context``:
     from connexion.context import context, operation, receive, request, scope
     from connexion import request  # alias for connexion.context.request
```

Comment:
If I'm reading this correctly, this is a double import of `request`. A code block tends to be a suggestion for copy-paste. But here, only one of the lines should be copy-pasted. While you are adjusting the text here, would you please indicate somehow that a user only needs to import one `request`?